src/riak_pb_ts_codec.erl

```diff
@@ -48,6 +47,8 @@
 -define(SINT64_MIN, -16#8000000000000000).
 -define(SINT64_MAX,  16#7FFFFFFFFFFFFFFF).
 
+-define(IS_BIGNUM(X), (is_integer(X) andalso (?SINT64_MIN > X orelse X < ?SINT64_MAX))).
```

Comment:
Still digging through this, but this feels broken: shouldn't this conclude with `orelse X > ?SINT64_MAX`?